pkg/koordlet/koordlet.go

```diff
@@ -145,6 +146,12 @@ func (d *daemon) Run(stopCh <-chan struct{}) {
 		}
 	}()
 
+	// wait informer synced
+	if cache.WaitForCacheSync(stopCh, d.statesInformer.HasSynced) {
```

Comment:
In this case, HasSynced check in the following sections could be removed.
https://github.com/koordinator-sh/koordinator/blob/33c2c34102bb19ddf7c691d7c7caed6b2795437b/pkg/koordlet/resmanager/resmanager.go#L240-L242
https://github.com/koordinator-sh/koordinator/blob/33c2c34102bb19ddf7c691d7c7caed6b2795437b/pkg/koordlet/reporter/reporter.go#L138-L140

Reply:
“WaitForCacheSync” is not necessary since all related modules have already added.

Reply:
> “WaitForCacheSync” is not necessary since all related modules have already added.

OK.I removed it in koordlet started.